x-pack/dockerlogbeat/pipelinemanager/pipelineManager.go

```diff
@@ -89,20 +109,90 @@ func (pm *PipelineManager) CreateClientWithConfig(containerConfig ContainerOutpu
 
 	reader, err := pipereader.NewReaderFromPath(file)
 	if err != nil {
-		return nil, errors.Wrap(err, "")
+		return nil, errors.Wrap(err, "error creating reader for docker log stream")
+	}
+
+	// Why is this empty by default? What should be here? Who knows!
```

Comment:
Hm... where does `info` come from? Use `debug.PrintStack()` if you want to find out.